_data/datasets/performance-expenditure-review/index.yaml

```diff
@@ -56,19 +56,26 @@ datasets:
   category:
     name: Performance Expenditure Review
     slug: performance-expenditure-review
-  created: '2018-07-17T12:21:46.369700'
-  description: The main purpose of this performance and expenditure review (PER) was
-    to understand how much is being spent on provincial roads, what drives this expenditure,
-    and what is being delivered. The aim was to identify where spending might be inefficient,
-    which could inform policy decisions and improve value for money. The PER aimed
-    to analyse the unit cost per kilometre of provincial roads, taking into account
-    the different costs structures across provinces due to topographical, climatic,
-    soil and other differences.
-  last_updated: '2018-07-17T12:21:52.851549'
+  created: '2018-07-17T12:13:41.726707'
+  description: "The biofuels regulatory framework aims to encourage the production\
+    \ of biofuel on a commercial scale by establishing transparent criteria for selecting\
```

Comment:
What is the intent behind the `\`s?

Reply:
Good question. The \ at the end of the line says it's not the end of the value. The \ at the beginning of the line says this space isn't indentation, it's part of the value.

The values haven't changed - it's the same as where it was before sorting. https://github.com/OpenUpSA/static-budget-portal/pull/361/files#diff-666db80f61581f9995674a66c06e40f3L860